db/app.test.js

```diff
@@ -26,6 +25,56 @@ describe("GET /api/topics", () => {
             })
     }) 
 })
+describe("GET /api/articles", () => {
+    test("200 responds with all the articles", () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body: {articles} }) => {
+                expect(articles).toHaveLength(13)
+                    expect(articles[5]).toEqual({
+                        article_id: 6,
+                        title: "A",
+                        topic: "mitch",
+                        author: "icellusedkars",
+                        body: "Delicious tin of cat food",
```

Comment:
There should be no body key and there should be a comment count key

Reply:
in the learn to code it says it should have a body key, can you show me what it should look like please?